running-requests/prompt-blueprints.mdx

```diff
@@ -126,27 +126,28 @@ print(response)
 
 ## Multi-Modal Variables
 
-The `image_variable` attribute allows you to dynamically insert a list of images into prompt template messages.
+PromptLayer supports any number of modalities in a single prompt. You can include text, images, videos, and other media types in your prompt templates.
 
-The `image_variable` is nested within the message content. The `type` and `image_url` are required fields specifying the type of content and the URL of the image, respectively. The `image_url` needs to be blank and is there for type requirements. The `image_variable` is the name of the list of images to be dynamically inserted.
+The `media_variable` content allows you to dynamically insert a list of medias into prompt template messages.
+
+The `media_variable` is nested within the message content. The `type` and `name` are required fields specifying the type of content and the name of the variable, respectively. The `name` is the name of the list of medias to be dynamically inserted.
 
 ```json
 {
     "role": "user",
     "content": [
         {
-            "type": "image_url",
-            "image_url": {"url": ""},
-            "image_variable": "images_to_insert"
+            "type": "media_variable",
+            "name": "images_to_insert"
```

Comment:
@abubakarsohail is this missing the actual media type? Where do i put in the public url or base64? 

Where is that described?